src/bot.py

```diff
@@ -329,6 +329,7 @@ def on_maintenance(self):
                 items = personal.inspect_items(game_stats=self._game_stats)
         keep_items = any([item.keep for item in items]) if items else None
         sell_items = any([item.sell for item in items]) if items else None
+        stash_gold = personal.get_inventory_gold_full()
```

Comment:
the inventory full of gold flag is overwritte if we buy any potions. And technically, it is right. We are not full of gold anymore... But we probably still want to stash gold : )